Dexterous-Rambutan/battle-code
# Deploy docker3

#### What's this PR do?

Alternate method of adding a Postgres container using the postgres image
#### What are the important parts of the code?

docker-compose.yml
server/helpers/dbConfig.js
#### How should this be tested by the reviewer?

Head over to http://104.131.141.22/ and try to hit /api/resetDB
Should expect a blank page, to load within 1 second
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

Refs #47 
#### Screenshots (if appropriate)
